pubsublite/spark-connector/README.md

```diff
@@ -0,0 +1,184 @@
+   export PUBSUBLITE_LOCATION=your-location
+
+   gcloud pubsub lite-topics create $TOPIC_ID \
+     --location=$PUBSUBLITE_LOCATION \
+     --partitions=1 \
+     --per-partition-bytes=30GiB
+
+   gcloud pubsub lite-subscriptions create $SUBSCRIPTION_ID \
+      --location=$PUBSUBLITE_LOCATION \
+      --topic=$TOPIC_ID
+   ```
+
+1. Create a Cloud Storage bucket.
+
+   ```bash
+   export BUCKET_ID=your-gcs-bucket-id
+
+   gsutil mb gs://$BUCKET_ID
+   ```
+
+## Python setup
+
+1. [Install Python and virtualenv].
+
+1. Clone the `python-docs-samples` repository.
+
+    ```bash
+    git clone https://github.com/GoogleCloudPlatform/python-docs-samples.git
+    ```
+
+1. Navigate to the sample code directory.
+
+    ```bash
+    cd python-docs-samples/pubsublite/spark-connector
+    ```
+
+1. Create a virtual environment and activate it.
+
+    ```bash
+    python -m venv env
+    source env/bin/activate
+    ```
+   > Once you are finished with the tutorial, you can deactivate
+   > the virtualenv and go back to your global Python environment
+   > by running `deactivate`.
+
+1. Install the required packages.
+    ```bash
+    python -m pip install -U -r requirements.txt
+    ```
+
+## Creating a Spark cluster on Dataproc
+
+1. Go to [Cloud Console for Dataproc].
+
+1. Go to Clusters, then [Create Cluster].
+   > **Note:** When setting up the cluster, you must choose
+   > [Dataproc Image Version 1.5] under ___Versioning___ because
+   > the connector currently only supports Spark 2.4.8.
+   > Additionally, in ___Manage security (optional)___, you
+   > must enable the cloud-platform scope for your cluster by
+   > checking "Allow API access to all Google Cloud services in
+   > the same project" under ___Project access___.
+
+   Here is an equivalent example using a `gcloud` command, with an additional optional argument to enable component gateway:
+
+    ```sh
+    export CLUSTER_ID=your-cluster-id
+    export DATAPROC_REGION=your-dataproc-region
+
+    gcloud dataproc clusters create $CLUSTER_ID \
+      --region $DATAPROC_REGION \
+      --image-version 1.5-debian10 \
+      --scopes 'https://www.googleapis.com/auth/cloud-platform' \
+      --enable-component-gateway
+    ```
+
+## Writing to Pub/Sub Lite
+
+[spark_streaming_to_pubsublite_example.py](spark_streaming_to_pubsublite_example.py) creates a streaming source of consecutive numbers with timestamps for 60 seconds and writes them to a Pub/Sub topic.
+
+To submit a write job:
+
+```sh
+export PROJECT_NUMBER=$(gcloud projects list --filter="projectId:$PROJECT_ID" --format="value(PROJECT_NUMBER)")
+
+gcloud dataproc jobs submit pyspark spark_streaming_to_pubsublite_example.py \
+    --region=$DATAPROC_REGION \
+    --cluster=$CLUSTER_ID \
+    --jars=gs://spark-lib/pubsublite/pubsublite-spark-sql-streaming-LATEST-with-dependencies.jar \
+    --driver-log-levels=root=INFO \
+    --properties=spark.master=yarn \
+    -- $PROJECT_NUMBER $PUBSUBLITE_LOCATION $TOPIC_ID
```

Comment:
I would recommend being explicit like --project_number=$PROJECT_NUMBER

Reply:
This is not required for Python [`argparse`](https://docs.python.org/3/library/argparse.html).

Reply:
I understand that it's not required, but I would prefer to be explicit here otherwise it could be wrong ordered.

Reply:
seems that the comment was lost, I would much prefer to be explicit here otherwise it could be wrong ordered.

Reply:
Updated to not use positional args but explicit args.